src/app/pages/list/action-list/action-list.component.html

```diff
@@ -0,0 +1,37 @@
+<mat-toolbar color="primary">
+    <div>Action List</div>
+    <div fxFlex style="margin: auto;"></div>
+    <button mat-icon-button (click)="onAddItem()"><mat-icon>add</mat-icon></button>
+</mat-toolbar>
+<mat-nav-list *ngIf="data.length > 0">
+    <pxb-info-list-item
+        [hidePadding]="true"
+        *ngFor="let i of data"
+        [class.selected]="isSelected(i)"
+        (click)="onSelected(i)"
+    >
+        <span pxb-title>{{ i.name }}</span>
+        <span pxb-subtitle>{{ i.details }}</span>
+        <span pxb-right-content>
+            <button
+                data-cy="action-menu"
```

Comment:
might be able to leave these off since we aren't doing cypress testing yet, but it doesn't hurt to leave it